### PR TITLE
Dao cont prettified

### PR DESCRIPTION
**Pull request by tharmaman**

### CODE REVIEW

File: src/foam/nanos/auth/Country.js [hunk 1]
```diff
@@ -12,6 +12,51 @@ foam.CLASS({
 
   ids: ['code'],
 
+  axioms: [
```
Comment (mcarcaso) [inline src/foam/nanos/auth/Country.js]: Exclude these

File: src/foam/u2/view/IconChoiceView.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
```
Comment (mcarcaso) [inline src/foam/u2/view/IconChoiceView.js]: Use this
```
/**
 * @license
 * Copyright 2019 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */
```
here and in all other new files.

File: src/foam/u2/search/Toolbar.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+foam.CLASS({
```
Comment (mcarcaso) [inline src/foam/u2/search/Toolbar.js:7]:
```
/**
 * @license
 * Copyright 2019 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */
```

File: src/foam/nanos/menu/DAOMenu2.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
```
Comment (kgrgreer) [inline src/foam/nanos/menu/DAOMenu2.js]: 2019

File: src/cronjobs [hunk 5]
```diff
@@ -1,3 +0,0 @@
-p({"class":"foam.nanos.cron.Cron","minute":1,"hour":-1,"dayOfMonth":-1,"month":-1,"dayOfWeek":-1,"scheduledTime":"2017-11-03T15:01:00.0Z","id":"sessionExpiry","description":"Remove expired sessions.","lastRun":"2017-11-03T14:21:09.231Z","server":true,"scheduled":false,"code":"EXPIRY_AGE = 8*3600*1000; /* 8 hours */ dao = x.get(\"localSessionDAO\");\ndao.where(foam.mlang.MLang.LT(foam.nanos.session.Session.LAST_USED, new Date(System.currentTimeMillis()-EXPIRY_AGE))).removeAll();","output":"","notes":"","enabled":true})
```
Comment (mcarcaso) [inline src/cronjobs]: I don't know if we need any of cronjobs. might be safest not to delete this.

File: src/foam/nanos/controller/ApplicationController.js [hunk 6]
```diff
@@ -106,7 +106,7 @@ foam.CLASS({
       font-size: 14px;
       letter-spacing: 0.2px;
       color: #373a3c;
-      background: %BACKGROUNDCOLOR%;
+      background: #f5f7fa;
```
Comment (mcarcaso) [inline src/foam/nanos/controller/ApplicationController.js]: undo

File: src/foam/u2/ActionView.js [hunk 7]
```diff
@@ -166,18 +166,18 @@ foam.CLASS({
      */
 
     ^tertiary {
-      border-color: #355bc4;
-      background-color: %SECONDARYCOLOR%;
-      color: white;
+      border: 2px solid transparent;
+      background-color: transparent;
+      color: #5e6061;
```
Comment (mcarcaso) [inline src/foam/u2/ActionView.js]: Shouldn't this be something from the look and feel?

File: src/foam/u2/ActionView.js [hunk 8]
```diff
@@ -166,18 +166,18 @@ foam.CLASS({
...
-      border-color: #294798;
-      background-color: %SECONDARYHOVERCOLOR%;
+      border-color: transparent;
+      background-color: transparent;
     }
 
-    ^tertiary:focus {
-      box-shadow: 0 1px 2px 0 rgba(22, 29, 37, 0.1), inset 0 1px 0 1px rgba(255, 255, 255, 0.06);
+    ^tertiary:focus:not(:hover) {
+      border-bottom-color: #4a33f4;
```
Comment (mcarcaso) [inline src/foam/u2/ActionView.js]: Same here. Probably should be part of a theme.

**Pull request closed without merging**